### DISCUSSION TITLE
Date field style - Is it possible that UI format for date field can have calendar style instead of 3 dropdowns for month, year and date?

### DISCUSSION THREAD

**Discussion by pchebro** [directus/directus#7573] [Archive]
Hi, 
We are using Directus 9.0.0-rc.64 version. Is it possible that UI format for date field can have calendar style instead of 3 dropdowns for month, year and date? Or does this feature already exists in other versions that are already there.

**Comment by rijkvanzanten** [comment]
> Is it possible that UI format for date field can have calendar style instead of 3 dropdowns for month, year and date?

Not yet, though there are other discussions around that exact idea, here's one: https://github.com/directus/directus/discussions/3728

**Reply by pchebro** [reply]
Was just reading some documentation and curious if it's possible to build our own custom interface in cms app and add that to date field in UI using below instructions? 
https://docs.directus.io/guides/interfaces/

**Reply by rijkvanzanten** [reply]
Yeah totally! That's always a possibility 🙂
